mesonbuild/compilers/compilers.py

```diff
@@ -55,7 +55,7 @@
 # This means we can't include .h headers here since they could be C, C++, ObjC, etc.
 # First suffix is the language's default.
 lang_suffixes = {
-    'c': ('c',),
+    'c': ('c'),
```

Comment:
I think this should be a tuple, not a string.

Reply:
indeed, that coma is intentional, python syntax weirdness.

Reply:
Fixed